pkg/apis/core/types.go

```diff
@@ -5506,6 +5535,12 @@ type SecurityContext struct {
 	// Note that this field cannot be set when spec.os.name is windows.
 	// +optional
 	SeccompProfile *SeccompProfile
+	// The apparmor options to use by this container. If apparmor options are
+	// provided at both the pod & container level, the container options
+	// override the pod options.
+	// Note that this field cannot be set when spec.os.name is windows.
```

Comment:
_aside_
```suggestion
	// Note that this field cannot be set when spec.os.name is "windows".
```
(and change other occurrences)